fl4health/parameter_exchange/parameter_packer.py

```diff
@@ -51,7 +51,10 @@ def unpack_parameters(self, packed_parameters: NDArrays) -> Tuple[NDArrays, floa
         # The last entry is extra packed fedprox variable
         split_size = len(packed_parameters) - 1
         model_parameters = packed_parameters[:split_size]
-        extra_fedprox_variable = float(packed_parameters[split_size:][0])
+        # The packed contents should have length 1
+        packed_contents = packed_parameters[split_size:]
+        assert len(packed_contents) == 1
```

Comment:
Just adding an extra sanity check in case this catches something for us in the future